server/src/test/java/org/elasticsearch/cluster/routing/allocation/allocator/DesiredBalanceStatsTests.java

```diff
@@ -60,9 +64,12 @@ public void testToXContent() {
                           "computation_iterations" : %d,
                           "computed_shard_movements" : %d,
                           "computation_time_in_millis" : %d,
-                          "reconciliation_time_in_millis" : %d
+                          "reconciliation_time_in_millis" : %d,
+                          "unassigned_shards" : %d,
+                          "total_allocations" : %d,
+                          "undesired_allocations" : %d,
+                          "undesired_allocations_fraction" : %s
```

Comment:
I wonder if it is also worth adding boolean `shard_allocation_reconciled`?

Reply:
I guess 0% would mean reconciled, right?

Reply:
Correct